core/src/main/java/com/microsoft/applicationinsights/channel/concrete/inprocess/InProcessTelemetryChannel.java

```diff
@@ -126,12 +124,17 @@ public InProcessTelemetryChannel() {
      *                                   Note, value should be between MIN_MAX_TELEMETRY_BUFFER_CAPACITY and MAX_MAX_TELEMETRY_BUFFER_CAPACITY inclusive
      */
     public InProcessTelemetryChannel(String endpointAddress, boolean developerMode, int maxTelemetryBufferCapacity, int sendIntervalInMillis) {
+        this(endpointAddress, null, developerMode, maxTelemetryBufferCapacity, sendIntervalInMillis, true);
+    }
+
+    public InProcessTelemetryChannel(String endpointAddress, String maxTransmissionStorageCapacity, boolean developerMode,
```

Comment:
@gavlyukovskiy this means we need this public constructor to be introduced in the channel? Is there a way to avoid this? I see that everything else is not configurable without this constructor like throtlling etc parameters. If you it is needed to have a constructor like this we need it's tests also.

Reply:
Added test with default values that are coming from spring-boot.